### PR TITLE
Add the instructions to install erlang on openSUSE and SLES distributions

### PR DESCRIPTION
**Pull request by Gsantomaggio**
- Add the instructions to install erlang on openSUSE and SLES 
- Fix the python documentation dependency lib, driver.py uses `python3`

### CODE REVIEW

File: site/install-rpm.xml [hunk 1]
```diff
@@ -190,6 +193,29 @@ yum install erlang
         </p>
       </doc:subsection>
 
+      <doc:subsection name="install-from-suse-repository">
+        <doc:heading>Erlang packages from openSUSE</doc:heading>
+        <p>
+          Install Erlang directly using zypper as root:
```
Comment (aojea) [inline site/install-rpm.xml]: I guess is better to recommend using sudo instead running as root

File: site/install-rpm.xml [hunk 2]
```diff
@@ -190,6 +193,29 @@ yum install erlang
...
+<pre class="sourcecode bash">
+zypper in erlang
```
Comment (aojea) [inline site/install-rpm.xml]: `sudo zypper install erlang` better to use the full subcommand instead of the abrev

File: site/install-rpm.xml [hunk 3]
```diff
@@ -190,6 +193,29 @@ yum install erlang
...
+
+          The version may be out-of-date, add the openSUSE 
+          factory <a href="http://download.opensuse.org/repositories/devel:/languages:/erlang:/Factory/"> repositories</a> 
+          to get the last version. For example for leap 15:
+<pre class="sourcecode bash">
+# add the openSUSE erlang factory repository for leap 15
+zypper addrepo http://download.opensuse.org/repositories/devel:/languages:/erlang:/Factory/openSUSE_Leap_15.0/ openSUSE-Erlang-Factory
+
+# import the key and refresh the repository
+zypper --gpg-auto-import-keys refresh
```
Comment (aojea) [inline site/install-rpm.xml]: `sudo zypper`

File: site/install-rpm.xml [hunk 4]
```diff
@@ -190,6 +193,29 @@ yum install erlang
...
 
+      <doc:subsection name="install-from-suse-repository">
+        <doc:heading>Erlang packages from openSUSE</doc:heading>
+        <p>
+          Install Erlang directly using zypper as root:
+<pre class="sourcecode bash">
+zypper in erlang
+</pre>
+
+          The version may be out-of-date, add the openSUSE
```
Comment (aojea) [inline site/install-rpm.xml]: I would rephrase as 

> Erlang versions distributed on the standard repositories are stable and meant for production uses. However, if you want to use the last version with the newest features you have to add the Factory repository

File: site/install-rpm.xml [hunk 5]
```diff
@@ -190,6 +193,29 @@ yum install erlang
...
+          to get the last version. For example for leap 15:
+<pre class="sourcecode bash">
+# add the openSUSE erlang factory repository for leap 15
+zypper addrepo http://download.opensuse.org/repositories/devel:/languages:/erlang:/Factory/openSUSE_Leap_15.0/ openSUSE-Erlang-Factory
+
+# import the key and refresh the repository
+zypper --gpg-auto-import-keys refresh
+
+# install the last erlang version
+zypper in erlang
```
Comment (aojea) [inline site/install-rpm.xml]: `sudo zypper install erlang`

**Comment by michaelklishin** [comment]
Thank you! Good to finally see SUSE covered.
